recipe/sppo/main_sppo.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2024 Bytedance Ltd. and/or its affiliates
+# Copyright 2023-2024 SGLang Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Note that we don't combine the main with ray_trainer as ray_trainer is used by other main.
+"""
+
+import os
+
+import hydra
+import ray
+
+from verl.trainer.ppo.reward import load_reward_manager
+
+from .sppo_ray_trainer import RaySPPOTrainer
+
+
+def get_custom_reward_fn(config):
```

Comment:
i do not think get_custom_reward_fn is used in this recipe. shall we just remove it?